node/generate-with-together-ai/src/main.js

```diff
@@ -92,13 +91,12 @@ export default async ({ req, res, error }) => {
 
   // Upload image to Appwrite Storage and return URL
   if (req.bodyJson.type === 'image') {
-    const endpoint =
-      process.env.APPWRITE_ENDPOINT || 'https://cloud.appwrite.io/v1';
+    const endpoint = process.env.APPWRITE_FUNCTION_API_ENDPOINT;
 
     const client = new Client()
       .setEndpoint(endpoint)
-      .setKey(process.env.APPWRITE_API_KEY)
-      .setProject(process.env.APPWRITE_FUNCTION_PROJECT_ID);
+      .setProject(process.env.APPWRITE_FUNCTION_PROJECT_ID)
+      .setKey(req.headers['x-appwrite-key']);
```

Comment:
```suggestion
      .setKey(req.headers['x-appwrite-key'] ?? '');
```
lets make sure we stay consistent in all examples. I see default value being used in some places and not in others.

Reply:
If I remember correctly, I used it only in languages where it was nessessary